setup.cfg

```diff
@@ -39,7 +38,6 @@ console_scripts =
 dev =
     bumpver==2023.1129
     dunamai==1.19.0
-pre_commit =
```

Comment:
Seems easier to fold this into dev. I am still keeping tests separate for CI convenience.